File: remote/src/main/java/com/kelsos/mbrc/ui/views/AlbumTrackView.java
package com.kelsos.mbrc.ui.views;

import com.kelsos.mbrc.domain.Album;
import com.kelsos.mbrc.domain.Track;
import java.util.List;

public interface AlbumTrackView {
  void updateAlbum(Album album);

  void updateTracks(List<Track> tracks);
}

File: app/src/main/kotlin/com/kelsos/mbrc/ui/views/AlbumTrackView.kt
package com.kelsos.mbrc.ui.views

import com.kelsos.mbrc.domain.Album
import com.kelsos.mbrc.domain.Track

interface AlbumTrackView {
  fun updateAlbum(album: Album)

  fun updateTracks(tracks: List<Track>)

  fun showPlaySuccess()

  fun showPlayFailed()

  fun showTrackSuccess()

  fun showTrackFailed()
}
